gnosis/safe-react
# Set v1.3.0 as default version

# WARNING
I will leave this as draft because there are some issues with gas estimation. Adding more gas manually solves the issues but I want to have some time with backend team in order to check this

## What it solves
Resolves #2572

## How this PR fixes it
Set v1.3.0 as latest version for all networks. This opens the possibility to execute an upgrade from previous version through the interface.

## How to test it
Try to upgrade safes from v1.0.0 onward.
Check normal operation with v1.3.0 safes (shouldn't be surprises because it's already working on BSC and Polygon)
